plugins/node/instrumentation-dataloader/src/index.ts

```diff
@@ -15,4 +15,4 @@
  */
 
 export * from './types';
-export { DataloaderInstrumentation } from './instrumentation';
+export * from './instrumentation';
```

Comment:
Didn't we actually want to move away from `export * ...` if we didn't need it? https://github.com/open-telemetry/opentelemetry-js/issues/4186